packages/traits/index.ts

```diff
@@ -171,3 +172,35 @@ export interface StarryUIComponentDefinition {
  exampleSource?: string
  example(theme: StarryUITheme): HTMLElement
 }
+
+export interface StarryUICellDimensionsTrait {
+ type: 'cellDimensions'
+ height: number
+ width: number
+}
+
+export interface StarryUICellRenderTrait {
+ type: 'cellRender'
+ cellRender: (x: number, y: number) => HTMLElement
+}
+
+export function withCellDimensions(
+ height: number,
+ width: number
+): StarryUICellDimensionsTrait {
+ return {
+  type: 'cellDimensions',
+  height,
+  width,
+ }
+}
+
+export function withCellRender(
+ cellRender: (x: number, y: number) => HTMLElement
+): StarryUICellRenderTrait {
+ return {
+  type: 'cellRender',
+  cellRender,
+ }
+}
+
```

Comment:
You should install VSCode Prettier Extension and format this file https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code

Reply:
Done